### ISSUE TITLE
Builds failing

### ISSUE THREAD

**Issue by ccjjmartin** [aquifer/aquifer#135]
## Background:
- Node Version: v5.11.1
- Aquifer Version: 1.0.0-beta1
## Error:

```
Creating symlinks...
Symlinking profiles/meredith_ag => profiles/meredith_ag
Destination directory does not exist. Creating: /Users/developer/vms/mdp.drp.sfg/build/profiles
Symlinking drush => sites/all/drush
Destination directory does not exist. Creating: /Users/developer/vms/mdp.drp.sfg/build/sites/all
{ [Error: ENOENT: no such file or directory, mkdir '/Users/developer/vms/mdp.drp.sfg/build/sites/all']
  errno: -2,
  code: 'ENOENT',
  syscall: 'mkdir',
  path: '/Users/developer/vms/mdp.drp.sfg/build/sites/all' }
```
## Current workaround:
- In your aquifer project delete your vendor/drush directory and perform `aquifer build` again

**Comment by chasingmaxwell** [comment]
This seems to be an issue with site local drush. Another workaround is to move drush from dev-dependencies to dependencies in your composer.json. This prevents site local drush.

**Comment by kporras07** [comment]
I can confirm de issue, I used the @ccjjmartin's workaround for a while, but now I can confirm that @chasingmaxwell's workaround works great.

**Comment by chasingmaxwell** [comment]
This should be solved by #159, I'm going to close this in favor of tracking the site local drush issue there.
